service/src/main/resources/quips/insultibot/backhandedCompliments.json

```diff
@@ -273,5 +273,71 @@
     "tags":[],
     "targetableText":"[TARGET] puts the \"fun\" back in \"fungal infection\".",
     "targetableSsml":"[TARGET] puts the fun<break /> back in fungal infection."
+  },
+  {
+    "quipGroup":"PEOPLE_OF_WALMART",
+    "text":"You're famous! I saw your picture on the internet. I think it was on 'People of Walmart'.",
+    "ssml":"You're famous! I saw your picture on the internet. I think it was on 'People of Walmart'.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"COCKROACH",
+    "text":"You're surprisingly adaptable in adverse conditions. Just like a cockroach!",
```

Comment:
The same could be said for any of these. So long as we keep the two groups mostly balanced it should be okay. Right now we have 18 winsults, 25 backhand. So we could flip it to keep things balanced.